src/Generator/Generators/CSharp/CSharpTextTemplate.cs

```diff
@@ -1273,10 +1273,51 @@ public void GenerateVTable(Class @class)
 
             GenerateVTableClassSetup(@class, wrappedEntries);
 
+            GenerateVTableCleanup(@class, wrappedEntries);
+
             WriteLine("#endregion");
             PopBlock(NewLineKind.BeforeNextBlock);
         }
 
+        private void GenerateVTableCleanup(Class @class, IList<VTableComponent> wrappedEntries)
+        {
+            WriteLine("private static readonly Destructor Finalise = new Destructor();");
```

Comment:
This is a private local variable, please rename it to start with a small letter.